### DISCUSSION TITLE
Migrate from HTML to Markdown

### DISCUSSION THREAD

**Discussion by OlegGulevskyy** [wailsapp/wails#2085] [Show and tell]
Made a little tool for my team for internal use - to help them migrate their HTML documentation articles (stored on Zendesk) to Markdown (docusaurus). This requires HTML to be parsed into `.mdx` files. Each HTML file contains images that also need to be migrated to be statically linked in docusaurus. `Go`'s amazing goroutines feature allows to download those images extremely fast (we are talking about 1.2k of images).
Initially this was a CLI tool..

But because of Wails - it is possible to have intuitive UI for my team as they are non tech savy (too much) and it was done in a matter of a day, not even full. 
I think this shows the amount of efforts you put in, API is intuitive and simple, it's fast and light weight.
Honestly - bravo for the project, hopefully I can contribute to it in the long run as I will continue building stuff with it.

Some images
![image](https://user-images.githubusercontent.com/43781031/201487099-a9f3b0da-6753-43db-a01f-550d19785a8a.png)
![image](https://user-images.githubusercontent.com/43781031/201487127-b09643c5-1763-4a17-a8c7-ffcd58a20326.png)

**Comment by leaanthony** [comment]
What a fantastic story! This is exactly the kind of use case Wails was written for. If you ever write a blog post about this, let us know and we'll link it on the official site 🙏
